draft-crawford-acme-delegated-http.md

```diff
@@ -59,16 +68,19 @@ Delegated HTTP validation combines the following benefits from the existing DNS-
 
 # Delegated HTTP Validation
 
-This proposal extends the HTTP-01 challenge type to support delegation using a DNS record, similar to the approach used for DNS-01 challenges. The process is as follows:
+This proposal extends the http-01 challenge type to support delegation using a DNS record, similar to the approach used for dns-01 challenges. The process for validating example.com is as follows:
+
+1. **DNS Configuration**: The domain owner creates a DNS record for `_acme-challenge.example.com` pointing to a server capable of serving HTTP validation requests. For example:
+
+    ```
+_acme-challenge.example.com. CNAME validation-server.example.net.
```

Comment:
```suggestion
_acme-challenge.example.com. CNAME validation-server-example.net.
```
Spotting.com and .net might not be straightforward

Reply:
I'm not sure that matter, to be honest -- the "validation-server" part should be pretty clear that it's "not the same server."